core/common/src/Localization.ts

```diff
@@ -9,7 +9,7 @@
 /** Options for Localization
  * @public
  */
-interface LocalizationOptions {
+export interface TranslationOptions {
```

Comment:
isnt this a breaking change?

Reply:
sorry not this one, but the one where you rename something we were already exporting.

Reply:
Ah, yes that is a breaking change. Well, there's two I guess:
- I changed a type name, changing `LocalizationOptions` to `ITwinLocalizationOptions` (in `ITwinLocalization.ts`), which is optionally used in the `ITwinLocalization` constructor.
- I also changed the type of a parameter in multiple `ITwinLocalization` methods from `TOptionsBase` (comes from i18next) to `TranslationOptions` which is newly being exported from `Localization.ts`.

Reply:
`ITwinLocalizationOptions` has been un-renamed back to `LocalizationOptions`.

I've kept the name change and added export for `TranslationOptions` in `Localization.ts` since that doesn't break anything... but can still undo that too. Well, at least the export. It _should_ still be renamed as otherwise it shares a name with `LocalizationOptions`.

(I'm going to post this as a general comment too for more visibility.)